util/signing/providers/hashicorp-vault/src/hsm/mod.rs

```diff
@@ -86,7 +87,7 @@ where
 			&self.client,
 			self.mount_name.as_str(),
 			self.key_name.replace("/", "_").as_str(),
-			base64::encode(message).as_str(),
+			base64::engine::general_purpose::STANDARD.encode(message).as_str(),
```

Comment:
Could use the re-exports in `base64::prelude`.
This is to remove deprecation warnings, right? The top-level `base64::encode` uses this engine anyway.

Reply:
Yes, this is done to remove the deprecation warning. We should provide a util function somewhere to hide the ugliness.